src/logger.ts

```diff
@@ -0,0 +1,54 @@
+import * as winston from 'winston';
+import * as os from 'os';
+import * as Elasticsearch from 'winston-elasticsearch';
```

Comment:
Why did you used big letter?

Reply:
Blindly copying code from other services is bad and should be punishable by kitchen duties!